lib/Cargo.toml

```diff
@@ -74,3 +74,4 @@ tokio = { workspace = true, features = ["full"] }
 default = []
 vendored-openssl = ["git2/vendored-openssl"]
 watchman = ["dep:tokio", "dep:watchman_client"]
+test-fakes = []
```

Comment:
The name "test-fakes" comes from the `fake-editor` and `fake-diff-editor` in the jj-cli crate, so it doesn't make sense to use that name here. Maybe "testing" is good enough? I don't have any other suggestions anyway. You can still keep the "test-fakes" name in the jj-cli crate (or change it, if you prefer).

Reply:
OK, done.